setup.py

```diff
@@ -5,30 +5,29 @@
 from setuptools import setup
 import versioneer
 
-# NOTE: These are tested in `continuous_integration/travis/test_imports.sh` If
+# NOTE: These are tested in `continuous_integration/test_imports.sh` If
 # you modify these, make sure to change the corresponding line there.
 extras_require = {
-    "array": ["numpy >= 1.15.1", "toolz >= 0.8.2"],
-    "bag": [
-        "cloudpickle >= 0.2.2",
-        "fsspec >= 0.6.0",
-        "toolz >= 0.8.2",
-        "partd >= 0.3.10",
+    "array": [
+        "numpy >= 1.15.1",
     ],
+    "bag": [],
```

Comment:
Leaving this for backwards compat, but I don't feel strongly about it.

Reply:
I think leaving for backwards compat is a good thing to do. Could you add a comment saying that's why we're keeping it around (to lower the chance we unintentionally remove it in the future)